### PR TITLE
SZL tests update

### PR DESCRIPTION
**Pull request by swietlana**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 import com.wikia.webdriver.common.contentpatterns.MobileWikis;
 import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.annotations.DontRun;
 import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.RunOnly;
 import com.wikia.webdriver.common.core.api.ArticleContent;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.logging.Log;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java]: MINOR Remove this unused import 'com.wikia.webdriver.common.logging.Log'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [ForcedLoginTests.java#L13](https://github.com/Wikia/selenium-tests/blob/491414265bfea5534617f9598a64e0ad5803373c/src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java#L13): Remove this unused import 'com.wikia.webdriver.common.logging.Log'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java [hunk 2]
```diff
@@ -60,22 +60,28 @@ public void anonCanLogInViaUserLoginPageSzl() {
...
-      Thread.sleep(2000);
-    } catch (InterruptedException e) {
-      Log.log("Explicit wait in signing in", e, false);
-    }
     base.verifyUserLoggedIn(user.getUserName());
     assertTrue(base.isStringInURL(URLsContent.SPECIAL_WATCHLIST));
   }
 
+  @RunOnly(language = "szl")
+  @Test(invocationCount = 5)
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java]: Should we always incovate it 5 times?

File: src/test/java/com/wikia/webdriver/testcases/desktop/mcfootertests/NonEnAnonMixedContentFooterTests.java [hunk 3]
```diff
@@ -107,6 +207,19 @@ void isUserTakenToDiscussionsAfterClickOnViewAll() {
...
+  @RunOnly(language = "szl")
+  @Test
+  void isUserTakenToDiscussionsAfterClickOnViewAllSzl() {
+    DiscussionsPage discussions = new MixedContentFooter().openWikiMainPage()
+        .scrollToMCFooter()
+        .getDiscussionsCard()
+        .scrollToDiscussions()
+        .clickOnViewAllLinkInDiscussions();
+
+    Assertion.assertTrue(discussions.isDiscussions());
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/testcases/desktop/mcfootertests/NonEnAnonMixedContentFooterTests.java]: Would be awesome to renamed it to something more descriptive like `isDiscussionsPresent()` or similar

File: src/test/java/com/wikia/webdriver/testcases/desktop/mcfootertests/NonEnAnonMixedContentFooterTests.java [hunk 4]
```diff
@@ -133,6 +263,21 @@ public void userIsTakenToDiscussionsPostViewAfterClickOnPost() {
...
+  @Test
+  public void userIsTakenToDiscussionsPostViewAfterClickOnPostSzl() {
+    new MixedContentFooter().openWikiMainPage()
+        .scrollToMCFooter()
+        .getDiscussionsCard()
+        .scrollToDiscussions()
+        .clickDiscussionsPost();
+
+    String url = driver.getCurrentUrl();
+    Assertion.assertTrue(url.contains(".wikia.com/szl/d/"));
```
Comment (nikodamn) [inline src/test/java/com/wikia/webdriver/testcases/desktop/mcfootertests/NonEnAnonMixedContentFooterTests.java]: `url` is used only once, so I guess we could use it inline i.e. `driver.getCurrentUrl().contains()`

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [ForcedLoginTests.java#L13](https://github.com/Wikia/selenium-tests/blob/dd63f05cf9f2e56d5d5581c3db35da748ea960b5/src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java#L13): Remove this unused import 'com.wikia.webdriver.common.logging.Log'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [ForcedLoginTests.java#L13](https://github.com/Wikia/selenium-tests/blob/2a1b544ef26b84ac9a06586a061227346751c5d0/src/test/java/com/wikia/webdriver/testcases/desktop/auth/ForcedLoginTests.java#L13): Remove this unused import 'com.wikia.webdriver.common.logging.Log'. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**